src/plugins/workspace/public/utils.ts

```diff
@@ -301,11 +310,16 @@ export const mergeDataSourcesWithConnections = (
   directQueryConnections: DataSourceConnection[]
 ): DataSourceConnection[] => {
   const openSearchConnections = convertDataSourcesToOpenSearchConnections(dataSources);
-
-  return [
+  const dataConnections = dataSources
+    .filter((ds) => ds.type === DATA_CONNECTION_SAVED_OBJECT_TYPE)
+    .map((ds) => ({ ...ds, name: ds.id }));
```

Comment:
The type of data source connection will be displayed in the data source connection table. There will be a type filter at the top right of the data source connection table. There is a type property in the [definition of data source saved object](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/21f421849b7b58571b22c6c25267a683433c88d7/src/plugins/data_source/server/saved_objects/data_connection.ts#L23). Will it been displayed in the data source connection table? If yes, i'm prefer using `DataSourceConnectionType.DataConnection` to indicate the new `data-connection`;

Reply:
Updated.